pkg/create-aerospike-conf.go

```diff
@@ -136,6 +138,44 @@ func (initp *InitParams) createAerospikeConf() error {
 	return nil
 }
 
+func (initp *InitParams) createAerospikeOpensslAndFipsCnf() error {
```

Comment:
I think its better to embed these files in the binary instead of copying the contents of the files in the code